src/com/amazon/ion/benchmark/DataConstructor.java

```diff
@@ -390,18 +390,19 @@ public static String constructString(Map<String, ReparsedConstraint> constraintM
     }
 
     /**
-     * Generate unicode codepoint randomly.
+     * Generate unicode codepoint randomly which matches the character from [A-Z] and [a-z].
      * @return generated codepoint.
      */
     private static int getCodePoint() {
-        Random random = new Random();
-        int type;
-        int codePoint;
-        do {
-            codePoint = random.nextInt(DEFAULT_RANGE.get(1) - DEFAULT_RANGE.get(0) + 1) + DEFAULT_RANGE.get(0);
-            type = Character.getType(codePoint);
-        } while (type == Character.PRIVATE_USE || type == Character.SURROGATE || type == Character.UNASSIGNED);
-        return codePoint;
+        int index = ThreadLocalRandom.current().nextInt(20);
+        int randomIndex = ThreadLocalRandom.current().nextInt(26);
+        if (index >= 0 && index < 10) {
+            // Randomly generate the unicode of character from [A-Z].
+            return randomIndex + 65;
```

Comment:
Please store the magic numbers `65` and `97` as constants with comments to describe what they are.